src/WCSimPhysicsList.cc

```diff
@@ -280,9 +280,6 @@ void WCSimPhysicsList::ConstructOp(){
   // theCherenkovProcess->SetMaxBetaChangePerStep(10.0);
   // theCherenkovProcess->SetTrackSecondariesFirst(true);
 
-  G4OpticalSurfaceModel themodel = unified;
-  theBoundaryProcess->SetModel(themodel);   // deprecated in 4.9.6
```

Comment:
As far as we can tell, this function call was not actually doing anything since the actual model is set as a property of each individual surface (which we do), rather than being set globally.